<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>995</width>
    <height>738</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>main_mune</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 250, 210);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>181</width>
      <height>531</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QFrame" name="frame_3">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QTreeView" name="treeView">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>181</width>
          <height>531</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_business_xiaohao">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>131</width>
          <height>26</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 250, 210);</string>
        </property>
        <property name="text">
         <string>销号</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_find">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>610</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>查询</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_find">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>610</y>
      <width>121</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_comfirm">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>550</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>确认</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_add">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>550</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>添加</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_delete">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>550</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>删除</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_update">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>550</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>修改</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_back">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>550</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>返回</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>851</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QFrame" name="frame_2">
       <property name="styleSheet">
        <string notr="true">selection-color: rgb(0, 170, 255);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QPushButton" name="pushButton_business">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>20</y>
          <width>91</width>
          <height>26</height>
         </rect>
        </property>
        <property name="text">
         <string>业务</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_database">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>20</y>
          <width>80</width>
          <height>26</height>
         </rect>
        </property>
        <property name="text">
         <string>数据库</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frame_4">
    <property name="geometry">
     <rect>
      <x>199</x>
      <y>79</y>
      <width>791</width>
      <height>451</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>791</width>
       <height>451</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 255, 255);
background-color: rgb(255, 250, 210);
background-color: rgb(85, 255, 127);
background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_des">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>60</y>
       <width>80</width>
       <height>26</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 250, 210);</string>
     </property>
     <property name="text">
      <string>des</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_asc">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>60</y>
       <width>80</width>
       <height>26</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 250, 210);</string>
     </property>
     <property name="text">
      <string>asc</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_status">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>60</y>
       <width>151</width>
       <height>26</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">selection-color: rgb(255, 170, 0);</string>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QLineEdit" name="lineEdit_xiaohao">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>100</y>
      <width>181</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_xiaohao">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>100</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>查询</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_xiaohao">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>100</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>请输入体检单号:</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView_xiaohao">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>180</y>
      <width>791</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_remove">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>100</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>销号</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>140</y>
      <width>51</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>全选</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>140</y>
      <width>82</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>复选</string>
    </property>
   </widget>
   <zorder>frame_4</zorder>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>pushButton_find</zorder>
   <zorder>lineEdit_find</zorder>
   <zorder>pushButton_comfirm</zorder>
   <zorder>pushButton_add</zorder>
   <zorder>pushButton_delete</zorder>
   <zorder>pushButton_update</zorder>
   <zorder>pushButton_back</zorder>
   <zorder>horizontalLayoutWidget</zorder>
   <zorder>lineEdit_xiaohao</zorder>
   <zorder>pushButton_xiaohao</zorder>
   <zorder>label_xiaohao</zorder>
   <zorder>tableView_xiaohao</zorder>
   <zorder>pushButton_remove</zorder>
   <zorder>radioButton</zorder>
   <zorder>checkBox</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>995</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menutestdb">
    <property name="title">
     <string>迪安</string>
    </property>
   </widget>
   <addaction name="menutestdb"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
